src/domains/chart/utils/formatters.ts

```diff
@@ -126,7 +129,7 @@ export const useFormatters = ({
   } = attributes
 
   const legendFormatValueDecimalsFromMinMax = useCallback((newMin: number, newMax: number) => {
-    if (newMin === min && newMax === max) {
+    if (safeEqualCheck(newMin, min) && safeEqualCheck(newMax, max)) {
```

Comment:
Just a note, so it's not a left-to-right arguments order? Like (prevProps, nextProps), but the other way around?

Reply:
oh yeah, that's a good idea to reverse it, haven't though of it. fixed